website/content/en/docs/reference/configuration/sinks/mezmo.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: LogDNA
-description: Deliver log event data to [LogDNA](https://logdna.com)
+title: Mezmo (formerly LogDNA)
+description: Deliver log event data to [Mezmo](https://mezmo.com)
 kind: sink
 layout: component
-tags: ["logdna", "component", "sink", "logs"]
+tags: ["logdna", "mezmo", "component", "sink", "logs"]
 ---
```

Comment:
```suggestion
tags: ["logdna", "mezmo", "component", "sink", "logs"]
aliases: ["/docs/reference/configuration/sinks/logdna"]
---
```

ref: https://github.com/vectordotdev/vector/blob/master/website/content/en/docs/reference/configuration/sources/http_server.md?plain=1#L7

cc @jszwedko I don't think the alias in the cue file does the redirect, but this frontmatter does

Reply:
It would appear so. Another thing that would be nice to automate.